src/util/transition/ngbTransition.ts

```diff
@@ -15,6 +15,7 @@ export interface NgbTransitionOptions<T> {
 
 export interface NgbTransitionCtx<T> {
   transition$: Subject<any>;
+  stop: () => void;
```

Comment:
I think I would prefer this function to be called `complete: () => {}`, because it properly completes the transition...

WDYT?

Reply:
Yes, I prefer it too.